.github/workflows/unit-tests.yml

```
@@ -47,6 +47,12 @@ jobs:
             continue_on_error: false
             job_label_php: 'current'
             job_label_wordpress: 'latest'
+          - php: '8.0'
+            wordpress: '6.2.0'
```

WP doesn't publish archives for pre-release versions, so we need to check out Core from SVN. Using `nightly` accomplishes that:

```suggestion
            wordpress: 'nightly'
```